src/routes/_surfaces/workspace/explore/time-range-constructor/TimeRangeConstructor.svelte

```diff
@@ -0,0 +1,79 @@
+<!--
```

Comment:
I think this could be renamed to `TimeControls.svelte`, since this is primarily containing the `TimeRangeSelector` and `TimeGrainSelector` as well as logic around a reactive update.

Reply:
Much better name! Changed.